cernopendata/modules/fixtures/data/docs/cms-guide-luminosity-calculation/cms-guide-luminosity-calculation.md

```diff
@@ -0,0 +1,275 @@
+# How to calculate luminosity for CMS open data
+
+## What is this?
+
+This document provides instruction for how to calculate the luminosity information
+for CMS in the open data environment. This is done using `brilcalc`, which is a command line tool from the CMS Beam Radiation Instrumentation and Luminosity (BRIL) group.
+
+Before you continue, consider that some of the information that you wish to calculate is already available for [2010](http://opendata-dev.web.cern.ch/record/1050), [2011](http://opendata-dev.web.cern.ch/record/1051), and [2012](http://opendata-dev.web.cern.ch/record/1052) data. The information available includes:
```

Comment:
We will have new lumi info records for 2011 and 2012 (see #2883) with record numbers 1053 and 1054, respectively.  Maybe it is a good idea to point to the new ones already.

Reply:
Done